dev/build.image/profiles/webProfile10/features.xml

```diff
@@ -12,3 +12,28 @@
 <feature>requestTiming-1.0</feature>
 <feature>microProfile-6.0</feature>
 <feature>microProfile-6.1</feature>
+<feature>appSecurity</feature>
+<feature>beanValidation</feature>
+<feature>cdi</feature>
+<feature>ejbLite</feature>
+<feature>el</feature>
+<feature>jaspic</feature>
+<feature>jaxrs</feature>
+<feature>jaxrsClient</feature>
```

Comment:
```suggestion
<feature>concurrent</feature>
<feature>enterpriseBeansLite</feature>
<feature>expressionLanguage</feature>
<feature>faces</feature>
```
Just following the names you used for jakartaee10 where you used the new names instead of the old EE 8 names.